### DISCUSSION TITLE
LocalPositionAtSpeed using TweenSettings

### DISCUSSION THREAD

**Discussion by Kayan-Oti** [KyryloKuzyk/PrimeTween#131] [FAQ] [answered]
Hi, I just wanna know if there is a reason why that method is not public
static Tween LocalPositionAtSpeed([NotNull] UnityEngine.Transform target, TweenSettings<UnityEngine.Vector3> settings)

**Comment by KyryloKuzyk** [comment, marked as answer]
Hey! That method is private bacause the `TweenSettings` struct doesn't have a field to represent `speed`. Everything in PrimeTween is based on duration, so adding a new `speed` field to the TweenSettings struct didn't feel right. And reusing the duration field to store speed is counterintuitive. So I just decided to keep things simple and made that method private to prevent confusion.

Hope this answers the question :)
